docs/modules/defences/detector_evasion.rst

```diff
@@ -2,6 +2,11 @@
 ====================================
 .. automodule:: art.defences.detector.evasion
 
+Base Class
```

Comment:
Have you tested rebuilding the docs?

Reply:
Rebuilding the produces the correct updated docs.

Reply:
Thank you!